app/src/main/java/com/zeapo/pwdstore/model/PasswordEntry.kt

```diff
@@ -118,5 +120,7 @@ class PasswordEntry(content: String, private val totpFinder: TotpFinder = UriTot
             "id:",
             "identity:"
         )
+
+        private val EXTRA_CONTENT_FIELDS = USERNAME_FIELDS + UriTotpFinder.TOTP_FIELDS
```

Comment:
Imo we should be only looking for either password or TOTP fields specifically in the first line. That's because I don't really see a use case for a username-only entry, but I can definitely see this randomly breaking entries where the password happens to start with `id:`.

Reply:
Hm that does make sense. Will make the change.